### PR TITLE
Add jenkins pipeline metrics ingestion

### PR DESCRIPTION
**Pull request by banderous**

### CODE REVIEW

File: src/main/config.ts [hunk 1]
```diff
@@ -21,5 +21,6 @@ export const config = {
   appinsightsKey: process.env.APPLICATIONINSIGHTS_CONNECTION_STRING || vault['secrets']?.['dtsse']?.['AppInsightsConnectionString'],
   jiraToken: process.env.JIRA_TOKEN || vault['secrets']?.['dtsse']?.['jira-token'],
   githubToken: process.env.GITHUB_TOKEN || vault['secrets']?.['dtsse']?.['github-token'],
+  cosmosKey: process.env.COSMOS_KEY || vault['secrets']?.['dtsse']?.['cosmos-key'],
```
Comment (linusnorton) [inline src/main/config.ts:24]: this probably needs to be added to the helm chart too

Comment (linusnorton) [inline src/main/config.ts:24]: think this is needed in the helm chart

File: migrations/sqls/20221007143236-jenkins-metrics-down.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+drop schema jenkins cascade;
```
Comment (linusnorton) [inline migrations/sqls/20221007143236-jenkins-metrics-down.sql:1]: I believe you need to drop the tables before you drop the schema but I might be wrong. I guess if your test is passing then I am wrong.

File: sonar-project.properties [hunk 3]
```diff
@@ -2,7 +2,7 @@ sonar.projectName=dtsse :: dashboard-ingestion
 sonar.projectKey=dtsse-dashboard-ingestion
 sonar.language=typescript
 sonar.sources=src/main
-sonar.test.exclusions=test/coverage/**,**/*.test.ts,**/*.integration_test.ts
+sonar.test.exclusions=test/coverage/**,**/*.test.ts,**/*.integration-test.ts
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.exclusions=src/**/*.test.ts,src/**/*.integration_test.ts,src/main/query/*.ts,src/main/run.ts
+sonar.exclusions=src/**/*.test.ts,src/**/*.integration-test.ts,src/main/query/*.ts,src/main/run.ts,src/main/executor.ts,src/main/jenkins/*.ts
```
Comment (linusnorton) [inline sonar-project.properties:7]: we should add a unit test for executor. I will do it in a separate PR

**Review by linusnorton — APPROVED** [review]
I would like to rename test_data and test_support but I will do it in a separate PR. At some it's just me being a bit pedantic :)

**Comment by banderous** [comment]
I think the executor is covered by my integration test iirc? Thanks for
review

**Pull request merged**